src/vs/workbench/browser/parts/editor/breadcrumbsModel.ts

```diff
@@ -142,6 +143,16 @@ export class EditorBreadcrumbsModel {
 		this._disposables.add(this._configurationService.onDidChangeConfiguration(e => {
 			if (e.affectsConfiguration('breadcrumbs')) {
 				this._updateOutline(true);
+				return;
+			}
+			if (this._editor && this._editor.getModel()) {
+				const editorModel = this._editor.getModel() as ITextModel;
+				const languageName = editorModel.getLanguageIdentifier().language;
+
+				// checking for changes in the current language override config
+				if (e.affectsConfiguration(`[${languageName}]`)) {
+					this._updateOutline(true);
```

Comment:
@sandy081 Is this the only why of knowing if a language-overidden setting has changed? doesn't the check on line 144 cover this?

Reply:
Yeah, unfortunately the event only knows about root keys. I would be improving this when  I extend lang specific settings for extensions. I will keep a note of it and remove this.

Reply:
I'll include a comment explaining this as well.

Reply:
9497acd875493a3dd6d240ea401422b3cc797fda